src/main/resources/public/jsx/geo-dash-help.js

```diff
@@ -0,0 +1,357 @@
+                    </Collapsible>
+                    <Collapsible
+                        trigger={this.state.LngObject.to_move_resize}
+                        triggerTagName="h2"
+                    >
+                        <img
+                            src={this.props.documentRoot + "/img/change_widget_layout.gif"}
+                            className="previewImg"
+                            onClick={(e) => this.toggleHelpImage(e)}
+                        />
+                        <br className="previewbreak"/>
+                        <ol>
+                            {this.getLi(this.state.LngObject.drag_drop)}
+                            {this.getLi(this.state.LngObject.resize_by)}
+                            {this.getLi(this.state.LngObject.widgets_realtime)}
+                            {this.getLi(this.state.LngObject.view_rendered)}
+                        </ol>
+                        <br style={{clear:"both;"}}/>
+                    </Collapsible>
+                    <Collapsible
+                        trigger={this.state.LngObject.gee_image_asset_creation}
+                        triggerTagName="h2"
+                    >
+                        <br className="previewbreak"/>
+                        <iframe
+                            style={{width: "100%", height: "570px"}}
+                            src="https://www.youtube.com/embed/l57IhmduVBQ"
+                            frameBorder="0"
+                            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                            allowFullScreen></iframe>
+                        <br style={{clear:"both;"}}/>
+                    </Collapsible>
+                    <Collapsible
+                        trigger={this.state.LngObject.gee_imageCollection_asset_creation}
+                        triggerTagName="h2"
+                    >
+                        <br className="previewbreak"/>
+                        <iframe
+                            style={{width: "100%", height: "570px"}}
+                            src="https://www.youtube.com/embed/7eIvltgDbXw"
+                            frameBorder="0"
+                            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                            allowFullScreen></iframe>
+                        <br style={{clear:"both;"}}/>
+                    </Collapsible>
+                </div>
+                <p>
+                    <span>{this.state.LngObject.pre_sepal_link}</span> <a
+                        href="http://www.openforis.org/tools/sepal.html"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >OpenForis-SEPAL</a> <span>{this.state.LngObject.post_sepal_link}</span>
+                </p>
+            </div>
+        );
+    }
+
+    getLi = text => <li style={{overflowWrap: "break-word"}}> {text} </li>;
+
+    toggleHelpImage = event => {
```

Comment:
Accessing the DOM / document directly is not  the preferred way in react if we can help it.  This page is not so react heavy that it would matter much, but this is a great example of re-usable code.  If you can convert this into a stateful component <GrowableImage> we can use it on other help pages.  Shorthand code:

```
class GrowableImage
state={fullSize: false}
const imgStyle = this.state.fullSize ? {full size style} : {normal size style}
<img style={imgStyle} onClick={this.setState{ fullSize: !this.state.fullSize />
```